src/liftwing_api/models/liftwing_model.py

```diff
@@ -0,0 +1,38 @@
+import requests
+import json
+
+class LiftwingModel:
+    def __init__(self, base_url):
+        self.base_url = base_url
+        # bare url
+
+    def request(self, endpoint_url: str, revision_id: int):
```

Comment:
This can be an [abstract class ](https://docs.python.org/3/library/abc.html) and the request method can be an abstractmethod that has the following arguments:

- payload : dict
- method :str ="POST"
- headers : dict